src/components/SpruceForm/errors.ts

```diff
@@ -4,6 +4,11 @@ export const transformErrors = (errors: AjvError[]) =>
   errors
     .map((error) => {
       switch (error.name) {
+        case "enum":
+          return {
+            ...error,
+            message: "",
+          };
```

Comment:
I don't think `ui:hideError` works for what I need it to do, but adding this handled the selection errors that Chaya brought up. The errors that fall under this case are the `should be equal to one of the allowed values` errors